crates/proof-of-sql/src/sql/proof_exprs/literal_expr.rs

```diff
@@ -46,22 +46,20 @@ impl ProofExpr for LiteralExpr {
     #[tracing::instrument(name = "LiteralExpr::result_evaluate", level = "debug", skip_all)]
     fn result_evaluate<'a, S: Scalar>(
         &self,
-        table_length: usize,
         alloc: &'a Bump,
-        _accessor: &'a dyn DataAccessor<S>,
+        table: &Table<'a, S>,
     ) -> Column<'a, S> {
-        Column::from_literal_with_length(&self.value, table_length, alloc)
+        Column::from_literal_with_length(&self.value, table.num_rows().unwrap_or(0), alloc)
```

Comment:
This is the only spot where a default makes sense to me. If there are no columns, it's not obvious to me what the proper behavior here is. Probably `1` instead of `0`.

Also, please add a test to cover this case.